lib/terraform/client.rb

```diff
@@ -50,8 +50,8 @@ def output(name:)
     end
 
     def version
-      execute command: factory.version_command do |value|
-        return ::Terraform::Version.create value: value
+      @version ||= execute command: factory.version_command do |value|
```

Comment:
@RyanJarv actually, this logic needs to be adjusted. `#execute` is designed to yield the value, not return it.